frontend/packages/helm-plugin/src/topology/helm-data-transformer.ts

```diff
@@ -135,21 +135,21 @@ export const getHelmGraphModelFromMap = (
             getImageForIconClass(`icon-helm`);
           helmResources[key].push(uid);
           const data = createTopologyNodeData(resource, item, TYPE_HELM_WORKLOAD, nodeIcon);
-          typedDataModel.nodes.push(
+          typedDataModel.nodes?.push(
             getTopologyNodeItem(resource, TYPE_HELM_WORKLOAD, data, WorkloadModelProps),
           );
           const groups = getTopologyHelmReleaseGroupItem(resource, helmResourcesMap, secrets);
-          mergeGroups(groups, typedDataModel.nodes);
+          mergeGroups(groups, typedDataModel.nodes as NodeModel[]);
```

Comment:
node is always defined, do you think it would be nicer to do this in line 121, then remove the `?.push` and the `as NodeModel[]`?

```
      const typedDataModel: Required<Omit<Model, 'graph'>> = {
```